### PR TITLE
back to changing dots instead of unflatten

### PR DESCRIPTION
**Pull request by edewit**
fixes: #3022
and also fixes: https://github.com/keycloak/keycloak/issues/13416

### CODE REVIEW

**Comment by edewit** [comment]
This fix might be better then https://github.com/keycloak/keycloak-admin-ui/pull/3023 @lexcao want to give it a try?

File: src/util.test.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { convertFormValuesToObject, convertToFormValues } from "./util";
 
 vi.mock("react");
 
+const TOKEN = "🍺";
```
Comment (jonkoops) [inline src/util.test.ts:6]: Proost! :beers:

**Comment by lexcao** [comment]
Hi, @edewit 
This fix makes sense to me.
It replaces all fields of the form.
Thank you!

File: src/identity-providers/add/AddMapper.tsx [hunk 2]
```diff
@@ -166,8 +166,6 @@ export default function AddMapper() {
 
   const setupForm = (mapper: IdentityProviderMapperRepresentation) => {
     convertToFormValues(mapper, form.setValue);
-    form.setValue("config.attributes", JSON.parse(mapper.config.attributes));
```
Comment (jonkoops) [inline src/identity-providers/add/AddMapper.tsx:169]: This should still be parsed, no?

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: src/authentication/components/ExecutionConfigModal.tsx [hunk 3]
```diff
@@ -94,7 +80,7 @@ export const ExecutionConfigModal = ({
   );
 
   useEffect(() => {
-    if (configDescription) setupForm(configDescription, config);
+    if (configDescription) setupForm(config);
```
[the remark this answers is not in the document]
Comment (edewit) [reply src/authentication/components/ExecutionConfigModal.tsx:83]: I think this was a left over of a old implementation now the `configDescription` is used to in conjunction with the dynamic fields

**Review by jonkoops — APPROVED** [review]

**Comment by mortenjoenby** [comment]
Hi.
This is a critical issue to us as we are supporting several hundred realms across several Keycloak cluster, which we have just upgraded to 19.0.1 ...!
When can we expect 19.0.2 to be released - assuming this will contain the fix?
Is it an option to run with the old Admin Console until then?

Thanks,
Morten

**Pull request merged**